README.md

```diff
@@ -1,11 +1,12 @@
 Formatted Text
 =============
+An imperative approach to multi-line formatted text; its 
```

Comment:
something a bit more verbose, would smoothen the transition
"This explainer is split into three parts for ease of reading"

Reply:
Thanks! I made the whole first paragraph a little more verbose and descriptive.

Reply:
See: https://github.com/WICG/canvas-formatted-text/commit/ee6113391b5048638e5c58d4d9da60f022481387